packages/integrations/tailwind/README.md

```diff
@@ -47,22 +57,35 @@ export default {
   integrations: [tailwind()],
 }
 ```
+  
+Then, restart the dev server.
+</details>
 
-## Getting started
+## Usage
 
-Tailwind's utility classes should be ready-to-use with zero config, including [preprocessor setup](https://tailwindcss.com/docs/using-with-preprocessors) and [production optimization](https://tailwindcss.com/docs/optimizing-for-production). Head to the [Tailwind docs](https://tailwindcss.com/docs/utility-first) to learn all of the options and features available!
+When you install the integration, Tailwind's utility classes should be ready to go right away. Head to the [Tailwind docs](https://tailwindcss.com/docs/utility-first) to learn how to use Tailwind, and if you see a utility class you want to try, add it to any HTML element to your project!
 
-## Configuration
+### Configuring Tailwind
 
-### Custom Tailwind Config File
+If you used the Quick Install instructions and said yes to each prompt, you'll see a `tailwind.config.cjs` file in your project's root directory. Use this file for your Tailwind configuration changes. You can learn how to customize Tailwind using this file [in the Tailwind docs](https://tailwindcss.com/docs/configuration).
 
-Add your own `tailwind.config.(js|cjs|mjs)` file to the base of your project, and this integration will respect it. This can be useful for setting a [custom theme](https://tailwindcss.com/docs/configuration) or providing other configuration?
+If it isn't there, you add your own `tailwind.config.(js|cjs|mjs)` file to the root directory and the integration will use its configurations. This can be great if you already have Tailwind configured in aother project and want to bring those settings over to this one.
 
-### config.path
+### Configuring the Integration
 
-You can give a different config file location using this integration's `config.path` option. If `config.path` is relative, it will be resolved relative to the root. 
+The Astro Tailwind integration handles the communication between Astro and Tailwind and it has its own options. Change these in the `astro.config.mjs` file (_not_ the Tailwind configuration file) which is where your project's integration settings live.
 
-Changing `config.path` is well supported in Astro, but not recommended overall since it can cause problems with other Tailwind integrations, like the official Tailwind VSCode extension.
+<details>
+  <summary>config.path</summary>
```

Comment:
We can maintain the heading hierarchy/styling by setting a heading here I think. Does introduce some margin, so if that seems visually unappealing, maybe just `<strong>`?

```suggestion
  <summary><h4>config.path</h4></summary>
```

Reply:
If some of these concerns are about how this will look on npmjs.com, should we maybe merge and be our own guinea pig?  😅 

I can spend the today with this, and make quick changes as necessary. @Jutanium @delucis